crates/iota/src/name_commands.rs

```diff
@@ -110,6 +110,13 @@ pub enum NameCommand {
         /// The coin to use for payment. If not provided, selects the first coin
         /// with enough balance.
         coin: Option<ObjectID>,
+        /// The address or alias to which the domain will point. If the flag is
+        /// specified without a value, the current active address will be used.
+        #[arg(long, require_equals = true, default_missing_value = "", num_args = 0..=1)]
+        set_target_address: Option<String>,
+        /// Set the reverse lookup for the domain.
```

Comment:
I would at least add to the doc that this flag can work only if the `set_target_address` flag doesn't point to an address different than the sender.

Reply:
And if ever you feel like it, you could actually check it in the `if set_reverse_lookup` to avoid wasting gas

Reply:
Target address you can set to anything, but reverse lookup you can only set if you control the target address

Reply:
Just try it and you'll see that it fails

Reply:
I commented on the doc of reverse lookup and not on the field, maybe that confuses you a bit though